index.html

```diff
@@ -656,9 +662,12 @@ <h2>Dr. Putchong Uthayopas</h2>
                                             <a target="default" href="https://twitter.com/fofliajinal">
                                                 <i class="icon social_twitter"></i>
                                             </a>
-					    <a href="https://www.linkedin.com/in/jinalfoflia" target="_self">
-						    <i class="icon social_linkedin"></i>
-						</a>
+					    <a href="https://www.linkedin.com/in/jinalfoflia" target="default">
+						<i class="icon social_linkedin"></i>
+					    </a>
```

Comment:
Please Change only the required line. you have `+3-3` changes where it should be `+1-1`.